sdks/python/apache_beam/coders/coder_impl.py

```diff
@@ -551,6 +577,11 @@ def get_estimated_size_and_observables(self, value, nested=False):
                 elem, nested=True))
         estimated_size += child_size
         observables += child_observables
+      # TODO: (BEAM-1537) Update to use an accurate count depending on size and
+      # count, currently we are underestimating the size by up to 10 bytes
+      # per block of data since we are not including the count prefix which
+      # occurs at most once per 64k of data and is upto 10 bytes long. The upper
+      # bound of the underestimate is 10 / 65536 ~= 0.0153% of the actual size.
```

Comment:
Technically the upper bound is 2/128, when we have a 2-byte size for a 128-byte value. However, as this is only a size *estimate* that's probably OK.